hardhat.config.ts

```diff
@@ -71,6 +74,7 @@ const config: HardhatUserConfig = {
       ...sharedNetworkConfig,
       url: 'https://public-node.testnet.rsk.co',
       chainId: 31,
+      gasPrice: 40000,
```

Comment:
I'm not sure about setting the gas price here, if it's necessary for testnet, it's not going to be the same for mainnet? 

And setting this gas price could not affect if the tx can or It's going to be included in a block?

Reply:
I had to set the `gasPrice` because without it, I was receiving the following error: `transaction's gas price exceeds cap`. I think this is one of the features that were introduced in the recent versions of RSKj. Since this is related to the interaction with the RSKj node, I thought this should have been a global configuration.
I haven't changed that value in mainnet because I haven't had the need to interact with it. If we experience the same problem (very likely), we may add it later on. WDYT?

Reply:
I agree, we may need to wait to understand the behavior